logbook-json/src/main/java/org/zalando/logbook/json/JsonPathBodyFilters.java

```diff
@@ -154,7 +154,11 @@ public DocumentContext filter(final DocumentContext context) {
             DocumentContext result = context;
 
             for (final Operation operation : operations) {
-                result = operation.filter(result);
+                try {
+                    result = operation.filter(result);
+                } catch (Exception e) {
+                    log.trace("Composite filter operation could not complete, the following exception {} has been thrown", e.getClass());
+                }
```

Comment:
This is nice in that it handles the issue. I have two suggestions to improve it:

1. Adding all exceptions to a list, and logging them all at once (if the list is non-empty). This is useful in that it won't clutter the logs, should there be multiple exceptions.
2. If the user is expected to understand what "composite filter" means, it's fine. Otherwise, something about `JsonPath Body Filter` can be used instead.

Reply:
Thank you for the suggestion, added this in https://github.com/zalando/logbook/pull/1514/commits/4fb418f4d4e1dca5829fd239a4270f7da4c264d4